Source/WebKit/NetworkProcess/cocoa/NetworkTaskCocoa.mm

```diff
@@ -207,7 +207,7 @@ static bool shouldCapCookieExpiryForThirdPartyIPAddress(const WebCore::IPAddress
         , ageCapForCNAMECloakedCookies = crossThreadCopy(m_ageCapForCNAMECloakedCookies)
         , weakTask = WeakObjCPtr<NSURLSessionTask>(task())
         , firstPartyRegistrableDomainName = crossThreadCopy(RegistrableDomain { firstPartyURL }.string())
-        , debugLoggingEnabled = m_networkSession->networkStorageSession()->trackingPreventionDebugLoggingEnabled()]
+        , debugLoggingEnabled = m_networkSession->checkedNetworkStorageSession()->trackingPreventionDebugLoggingEnabled()]
```

Comment:
I believe we can inline `trackingPreventionDebugLoggingEnabled()` instead, it is a trivial function and we shouldn't need to call `checkedNetworkStorageSession()` here.

Reply:
Done.